backend/models/game.go

```diff
@@ -5,12 +5,11 @@ import (
 )
 
 type Game struct {
-	ID           uuid.UUID
-	Name         string
+	ID      uuid.UUID
+	Title        string
 	Description  string
 	UserID       uuid.UUID
-	CreationDate string
-	Version      string
+	VersionNumber string
```

Comment:
The game model will need to have a reference to where its game files are stored. For cloud storage this should just be a URL, so string should be fine.
```suggestion
	VersionNumber string
	GameStorageURL string
```